packages/backend/src/services/analyzer.test.ts

```
@@ -0,0 +1,300 @@
+    it("should cover critical security patterns", () => {
+      const allTags = TEST_CASES.flatMap((tc) => tc.tags);
+      const required = ["transfer", "approval", "high-risk", "scam-pattern"];
+
+      for (const tag of required) {
+        expect(allTags).toContain(tag);
+      }
+    });
+  });
+});
+
+export { TEST_CASES, type TestCase, type TestResult };
```

Biome (2.3.13)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)